File: kinny-user-interface/src/main/java/com/kinny/user/service/UserService.java
package com.kinny.user.service;

import com.kinny.pojo.TbUser;

/**
 * @author qgy
 * @create 2019/6/22 - 17:12
 */
public interface UserService {

    /**
     *  注册  验证验证码
     * @param user
     * @param smsCode
     */
    public void registry(TbUser user, String smsCode);

    /**
     * 发送验证码
     * @param phone
     */
    public void sendSmsCode(String phone);


}

File: kinny-user-web/src/main/webapp/js/service/UserService.js
app.service('UserService', function ($http) {

    this.registry = function (entity, smsCode) {
        return $http.post('user/registry.do?smsCode=' + smsCode, entity);
    }

    this.sendSms = function (phone) {
        return $http.get('user/sendSms.do?phone=' + phone);
    }


});